kiali-server/templates/role.yaml

```diff
@@ -99,4 +99,10 @@ rules:
   - list
   - patch
   - watch
+- apiGroups:
+  - authentication.k8s.io
```

Comment:
Maintain consistency with the rest of this file - specify apiGroups using `[ ]` notation.
```suggestion
- apiGroups: ["authentication.k8s.io"]
```

Reply:
done